### ISSUE TITLE
500 Internal Server Error with chat-ui

### ISSUE THREAD

**Issue by leemgs** [huggingface/chat-ui#1221]
I executed an inference server with the address http://192.168.0.185:7777/generate_stream using text-generation-inference (TGI) v.2.0.4. When executing commands with curl, the inference results are responding normally. For ease of use, I am going to use chat-ui.  Below is the .env.local file's content of chat-ui. 

```
$ vi .env.local
  1 MONGODB_URL=mongodb://127.0.0.1:27017
  2 HF_TOKEN=hf_***********************************
  3 ALLOW_INSECURE_COOKIES=true
  4 MODELS=`[
  5   {
  6     "name":"samsung-codellama3-70b-custom",
  7     "endpoints":[{"type":"tgi","url":"http://192.168.0.185:7777/generate_stream"}],
  8     "description":"A_Coding_Assistant_Model",
  9     "userMessageToken":"<|prompter|>",
 10     "assistantMessageToken":"<|assistant|>",
 11     "messageEndToken":"</s>",
 12     "preprompt":"It_is_an_LLM-based_AI_assistant."',
 13     "parameters":{
 14       "temperature":0.2,
 15       "top_p":0.9,
 16       "repetition_penalty":1.2,
 17       "top_k":10,
 18       "truncate":1000,
 19       "max_new_tokens":500
 20     }
 21   }
 22 ]`
```

Then, I run `$ docker run -p 3000:3000 --env-file .env.local -v chat-ui:/data --name chat-ui ghcr.io/huggingface/chat-ui-db` command. Unfortunately, when I visited http://localhost:3000 with the MS Edge web browser, I got the error “500: An error occurred” as shown below. 

* Screenshot:
![image](https://github.com/huggingface/chat-ui/assets/82404/6fec9357-8969-4b31-b657-a50bafad6114)

* log message:
`{"level":50,"time":1717033937576,"pid":30,"hostname":"c5e9372bf1c1","locals":{"sessionId":"f19bea94fb83ffe9b2aa5d9c3247d9dc1e819772e3b0b4557294cc9a7e884bf0"},"url":"http://localhost:3000/","params":{},"request":{},"error":{"lineNumber":1,"columnNumber":1},"errorId":"7b3df79b-b4d0-4573-b92d-4ba0c182828b"}`

I am wondering what could be causing this error.  Welcome to any hints to fix this issue.

#### References
* https://github.com/huggingface/chat-ui/issues?q=is%3Aissue+%22internal+server+error%22
* https://github.com/huggingface/chat-ui/blob/main/src/lib/server/models.ts#L198

**Comment by nsarrazin** [comment]
Not sure if that's the issue but on line 12 of your `.env.local` you have an extra `'` 

> ` 12     "preprompt":"It_is_an_LLM-based_AI_assistant."',`

should be 
>  ` 12     "preprompt":"It_is_an_LLM-based_AI_assistant.",`

**Comment by leemgs** [comment]
>  12     "preprompt":"It_is_an_LLM-based_AI_assistant.",

@nsarrazin , Thanks.  However, I got the same issue even though I removed the symbol `'`.  It's weird. 


* The log message: 
`{"level":50,"time":1717056763881,"pid":29,"hostname":"581477a135ef","locals":{"sessionId":"0146d80a16335d1f3e24b96e41a887f32277762604cbb0badc6ef74d215bc37f"},"url":"http://localhost:3000/","params":{},"request":{},"error":{"lineNumber":1,"columnNumber":1},"errorId":"49e8ceba-e970-42b2-ae5d-a8190fd98589"}  `
